src/ui/components/DropdownMenu/index.js

```diff
@@ -11,7 +11,8 @@ import './styles.scss';
 
 type Props = {|
   text: string,
-  children: Array<DropdownMenuItem>,
+  // TODO: Replace Array by React.ChildrenArray when we upgrade Flow to 0.53+.
```

Comment:
Ah, yeah, I saw your comment about that. I'll try to look at that upgrade soon. We have some things to fix.